src/compiler/factory/nodeFactory.ts

```diff
@@ -1188,7 +1188,7 @@ namespace ts {
         // @api
         function createDecorator(expression: Expression) {
             const node = createBaseNode<Decorator>(SyntaxKind.Decorator);
-            node.expression = parenthesizerRules().parenthesizeLeftSideOfAccess(expression);
+            node.expression = parenthesizerRules().parenthesizeLeftSideOfAccess(expression, /*optionalChain*/ false);

```

Comment:
This parameter is optional, but I see that it's been provided explicitly at all of these call sites even if false; is that intentional?

Reply:
Yes, this is intentional. The function is also passed as an argument to `emitExpression` in _emitter.ts_, which does not expect (and does not pass) a value, so it's not actually provided at all call sites due to an indirect invocation.